src/history/hs_rec.c

```diff
@@ -821,12 +822,11 @@ __wt_hs_insert_updates(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_MULTI *mult
 }
 
 /*
- * __wt_hs_delete_key_from_ts --
- *     Delete history store content of a given key from a timestamp and optionally reinsert them
- *     with ts-1 timestamp.
+ * __wt_hs_delete_key --
+ *     Delete history store content of a given key and optionally reinsert them with ts-1 timestamp.
```

Comment:
I think we just reinsert with 0 ts in this case.

Reply:
Updated in [b15e10c](https://github.com/wiredtiger/wiredtiger/pull/8199/commits/b15e10cf317554fe842d32699de8391b59db43b0)